app/src/main/java/org/schabi/newpipe/local/dialog/PlaylistAppendDialog.java

```diff
@@ -179,10 +183,69 @@ private void onPlaylistSelected(@NonNull final LocalPlaylistManager manager,
                     .subscribe(ignored -> successToast.show()));
         }
 
-        playlistDisposables.add(manager.appendToPlaylist(playlist.uid, streams)
-                .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(ignored -> successToast.show()));
+        /*Check if video had already added to this playlist*/
+        final StreamEntity videoToAdd = streams.get(0); //the first video is the target video to add
```

Comment:
Please format all your comments like this with a space between the comments symbols and the comment to increase readability:
```suggestion
        /* Check if video had already added to this playlist */
        final StreamEntity videoToAdd = streams.get(0); // the first video is the target video to add
```

Reply:
Why not just //?